packages/jest-phabricator-coverage/src/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+function formatResults(testResult, coverageMap) {
```

Comment:
So, thinking about this, I do think the `formatTestResults` function in jest-cli is based on this `formatResults` function but it has since gotten extended quite a bit. I think we should take `formatTestResults`, move it to `jest-util` like you proposed and throw away this function here. The formatResult function there already supports passing in a custom coverage mapping function.